minifi-toolkit/minifi-toolkit-configuration/src/main/java/org/apache/nifi/minifi/toolkit/configuration/ConfigMain.java

```diff
@@ -85,8 +85,6 @@ public static void main(String[] args) {
     public static void printValidateUsage() {
         System.out.println("Validate Usage:");
         System.out.println();
-        System.out.print("java ");
```

Comment:
Should this and the other java invocation remain given that the interpretation of arguments is handled by the script when invoked from the script?  This is valid in the context the class is run by itself.